src/views/tickets/BaseTicketList.vue

```diff
@@ -41,6 +42,72 @@ export default {
       loading: true,
       getDrawerTitle: () => ' ',
       createDrawer: () => import('@/views/tickets/RequestAssetPerm/CreateUpdate'),
+      quickFilters: [
+        {
+          label: this.$t('Type'),
+          options: [
+            {
+              label: this.$t('ApplyAsset'),
+              filter: {
+                type: 'apply_asset'
+              }
+            },
+            {
+              label: this.$t('LoginConfirm'),
+              filter: {
+                type: 'login_confirm'
+              }
+            },
+            {
+              label: this.$t('CommandConfirm'),
+              filter: {
+                type: 'command_confirm'
+              }
+            },
+            {
+              label: this.$t('LoginAssetConfirm'),
+              filter: {
+                type: 'login_asset_confirm'
+              }
+            }
+          ]
+        },
+        {
+          label: this.$t('Action'),
+          options: [
+            {
+              label: this.$t('All'),
+              filter: {
+                state: 'all'
+              }
+            },
+            {
+              label: this.$t('Open'),
+              filter: {
+                state: 'pending'
+              }
+            },
+            {
+              label: this.$t('Cancel'),
+              filter: {
+                state: 'closed'
+              }
+            },
+            {
+              label: this.$t('Approved'),
+              filter: {
+                state: 'approved'
+              }
+            },
+            {
+              label: this.$t('Rejected'),
+              filter: {
+                state: 'rejected'
+              }
+            }
+          ]
+        }
+      ],
       detailDrawer: null,
       ticketTableConfig: {
         url: this.url,
```

Comment:
Please share the actual context of the code you'd like checked for comparison against. Without that information, it is impossible to provide an accurate assessment or recommendations on potential issues or optimizations. Please refer back to the original text with more specific details about what parts of the code are being compared and how discrepancies might exist from your knowledge cutoff day in 2021.

For example:

```
<template>
  // Your code here
</template>

<script>
export default {}
</script>

<style></style>
```